src/components/citation/DocumentViewer.tsx

```
@@ -0,0 +1,509 @@
+  // Sanitize HTML to prevent XSS attacks
+  const displayContent = DOMPurify.sanitize(highlightedContent, { USE_PROFILES: { html: true } });
+
+  // Get orphaned citations from both the citations array and recentChanges
+  const orphanedFromCitations = citations.filter(c => c.isOrphaned);
+  const orphanedFromChanges = (recentChanges || [])
+    .filter(c => c.changeType === 'deleted' || (c.newNumber === null && c.oldText === c.newText))
+    .map(c => ({ rawText: c.oldText || c.newText || '' }));
+
+  // Combine and dedupe orphaned citations
+  const orphanedTexts = new Set<string>();
+  orphanedFromCitations.forEach(c => orphanedTexts.add(c.rawText));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)